tracing-futures/CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
-# 0.2.0 (Nov XX, 2019)
+# 0.2.1
+
+- docs: add `doc-cfg` attributes to tracing, tracing-core, tracing-futures, tracing-subscriber (#523, #525) 
```

Comment:
I don't think it's necessary to mention changes to other crates in the `tracing-futures` changelog. 

Also, for the other crates where we addded the doc-cfg attributes, I phrased this like 
```markdown
### Added

- ...
- API documentation now shows which features are required by feature-flagged items (#523)

Reply:
will add.